ent/README.md

```diff
@@ -0,0 +1,96 @@
+# Go2sky with Ent (https://entgo.io)
+
+## Installation
+
+```bash
+1. Define an schema type.
+2. Generate the specified Ent Model files.
+3. Apply it to your application.
+```
+
+## Steps
+```go
+go install entgo.io/ent/cmd/ent@v0.11.2
+ent generate --feature sql/upsert --target ./gen/entschema ./schema
+```
+
+## Usage
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/SkyAPM/go2sky"
+	"github.com/SkyAPM/go2sky/reporter"
+	"go2sky-plugins/ent/test/gen/entschema"
+	entuser "go2sky-plugins/ent/test/gen/entschema/user"
+	v3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
+	"time"
+)
+
+func main() {
+	ctx := context.Background()
+
+	r, err := reporter.NewGRPCReporter("127.0.0.1:11800", reporter.WithCheckInterval(time.Second*10))
+	if err != nil {
+		panic(err)
+	}
+	tracer, err := go2sky.NewTracer("example", go2sky.WithReporter(r))
+	if err != nil {
+		panic(err)
+	}
+
+	entClient := NewEntClient01(tracer) // entClient := NewEntClient02(tracer)
+
+	go func() {
+		span, ctx, err := tracer.CreateLocalSpan(ctx, go2sky.WithOperationName("test_ent"))
+		if err != nil {
+			return
+		}
+		defer span.End()
+		span.SetComponent(5012)
+		span.SetSpanLayer(v3.SpanLayer_Database)
+
+		user, err := entClient.User.Query().Where(entuser.ID(1)).First(ctx)
+		if err != nil {
+			return
+		}
+		fmt.Printf("%+v \n", user)
+
+		if err = entClient.User.Update().
+			SetName("test01").
+			Where(entuser.ID(1)).
+			Exec(ctx); err != nil {
+			return
+		}
+
+		user, err = entClient.User.Query().Where(entuser.ID(1)).First(ctx)
+		if err != nil {
+			return
+		}
+		fmt.Printf("%+v \n", user)
+
+		if err = WithTx(ctx, entClient, func(tx *entschema.Tx) error {
+			if err = tx.User.Create().
+				SetName("test02").
+				SetAge(12).
+				Exec(ctx); err != nil {
+				return err
+			}
+			return nil
+		}); err != nil {
+			return
+		}
+
+		if _, err = entClient.User.Delete().
+			Where(entuser.Name("test02")).
+			Exec(ctx); err != nil {
+			return
+		}
+	}()
+
+	select {}
+}
+```
+![img.png](img.png)
```

Comment:
I don't think we should keep img in the doc and GitHub repo. This could be easily expired, such as currently, v8(you provided) has not been maintained anymore. 

https://skywalking.apache.org/events/deprecate-v8/

Reply:
Please remove this link and v8 sceenshot.

Reply:
ok